### PR TITLE
feat: update readme with cli usage

### PR DESCRIPTION
**Pull request by spencerwooo**

### CODE REVIEW

**Comment by spencerwooo** [comment]
Currently blocked by upstream: <https://github.com/beetcb/ms-graph-cli/pull/1>. Also lacks Chinese translation and demo GIF screencast.

**Comment by beetcb** [comment]
Let me translate the tool into Chinese and add the command line option, also gonna fix demo's animations in beetcb/ms-graph-cli#2 in due time

**Comment by beetcb** [comment]
ms-graph-cli

Hi @spencerwooo 👋 How does this look, any suggestions?

**Comment by spencerwooo** [comment]
> ms-graph-cli
> 
> Hi @spencerwooo ,how does this look, any suggestions?

Perfect. :heart:

**Comment by spencerwooo** [comment]
@beetcb 俩 README 还有啥要补充的吗比如命令行参数和动图？

**Comment by beetcb** [comment]
我录个动图吧，等我先发布到 npm

**Comment by beetcb** [comment]
这不会造成卡顿了，之前那个可能是粘贴的重定向地址太长了

File: README-CN.md [hunk 1]
```diff
@@ -97,24 +97,12 @@ _又臭又长的中文版部署指南预警！同时感谢原项目的 Token 在
...
-      ![](assets/got-code.png)
-
-      点击 `OK`，并进行下一步操作：
-
-   3. 在 `5. Exchange Access Token` 处，我们应该有填好了的上一步骤中获取的 `Code`，这里只需要输入刚刚拿到的 `client_secret`：
-
-      ![](assets/get-access-token.png)
-
-      点击 `GET TOKEN`，如果我们遇到类似 `error: "invalid_request"` 的报错，**那么请参考 Issue [#13](https://github.com/spencerwooo/onedrive-cf-index/issues/13#issuecomment-671027672) 给出的解决方案。** 如果一切正常，那么拿到我们的 `access_token`，以及如果需要，在下一步 `Refresh Token` 处拿到我们的 `refresh_token`。
+   根据你自己的情况选择合适的选项，并输入我们上面获取到的一系列 token 令牌配置等，其中 `redirect_url` 可以直接设置为 `http://localhost`。
```
Comment (beetcb) [inline README-CN.md]: 可以直接留空，因为默认值是 http://localhost

Comment (beetcb) [reply README-CN.md]: `redirect_url`

Comment (spencerwooo) [reply README-CN.md]: 干，我手快了一步

Comment (beetcb) [reply README-CN.md]: 我按下 submit 就刚刚好 merge，哈哈哈

Comment (spencerwooo) [reply README-CN.md]: 没事 影响不大

**Pull request merged**